src/components/App.js

```diff
@@ -1,26 +1,16 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { createStore } from 'redux';
 import reducers from '../reducers/reducers';
 
-import { Routes } from '../helpers/routes';
+import Routes from '../helpers/routes';
 
-export class App extends Component {
-  constructor() {
-    super();
+function App() {
+  const store = createStore(reducers);
 
-    const store = createStore(reducers);
-
-    this.state = {
-      store,
-    };
-
-    store.subscribe(() => {
-      console.log('store.getState()', store.getState());
-    });
-  }
-  render() {
-    return <Routes />;
-  }
+  store.subscribe(() => {
```

Comment:
Does this work without storing `store` in a component state?

Reply:
Looking further down, I just noticed that we currently have two stores, and I don't think this one is being used, so we are probably fine to delete it. I would double check in case though. For reference, the other one is inside of the `Routes` component.

Reply:
Ya i just saw that too, and ya we dont need the state for App.js so i got rid of that.